src/scripts/migrate.py

```diff
@@ -154,6 +154,8 @@ def slurp_docs():
         ontology_name = os.path.basename(path).replace(FILENAME_GLOB_PATTERN[1:], '')
         ontology_page_relpath = f'./migrate_{ontology_name.lower()}.md'
         df = pd.read_csv(path, sep='\t').fillna('')
+        # Fix issue where the | in the `parents` column is causing table rendering issues
+        df['parents'] = df['parents'].apply(lambda x: x.replace('|', ','))
```

Comment:
## The bug fix
@twhetzel I ran this successfully w/out error. Let me know when you've reviewed, and I can run the full build for this per protocol.

Reply:
@twhetzel Forgot this was still open.

I have a comment saying:
>         # Fix issue where the | in the `parents` column is causing table rendering issues

To clarify, there was a jaggedness to the table that was being caused by this problem:
- Markdown tables use `|` as a cell delimiter
- In ROBOT templates, we usually use `|` as a delimiter

This collision was causing the Markdown tables to render multiple cells when there were multiple parents. You can see it if you look in some of the migrate `.md` files. This PR fixes that.